pkgs/by-name/ki/kittycad-kcl-lsp/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kittycad-kcl-lsp";
+  version = "0.1.61";
+
+  src = fetchFromGitHub {
+    owner = "KittyCAD";
+    repo = "kcl-lsp";
+    rev = "v${version}";
```

Comment:
`rev = "refs/tags/v${version}"` is safer if there is ever a branch named the same as the tag

Reply:
Thanks for the tip. I've made the change.